### ISSUE TITLE
Stopping the recorder closes the Browser

### ISSUE THREAD

**Issue by Rajagopalan-M** [kapoorlakshya/screen-recorder#91]
Stopping the recorder closes the browser. 

`@recorder.close`

closes the browser.

**Comment by kapoorlakshya** [comment]
@Rajagopalan-M Hmm... that's strange. There is no `#close` method, just `#stop` to stop recording.

And stopping the recording should not close the browser. In fact, it's the opposite - the recording will stop automatically if the browser (source window) is closed.

**Comment by Rajagopalan-M** [comment]
Hi, Nope, I am using @recorder.stop not @recorder.close. It's throwing the following error when I execute @recorder.stop

"the ffprobe binary could not be found" error is happening because I am giving setting up the path using this line "ScreenRecorder.ffmpeg_binary = File.expand_path('C:\A\RubyAutomation_POM\VideoRecorder\ffmpeg.exe')" So I am not setting anything in the enviromental variable. So your program is missing out the path for ffprobe. I think you have to include the path for ffprobe when I am setting the path of ffmpeg.exe.

```
[DevTools listening on ws://127.0.0.1:58990/devtools/browser/d76f07d9-311f-40ea-9ab6-d165cb563bc8

Uncaught exception: No such file or directory - the ffprobe binary could not be found in C:\Ruby26\bin;C:\Ruby26\bin;C:\Program Files\ImageMagick-7.0.10-Q16-HDRI;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\ProgramData\Oracle\Java\javapath;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Windows\CCM;C:\work\cit.1.5\maven\bin;C:\Software\TortoiseSVN\bin;C:\Android;C:\Windows\System32;C:\Program Files (x86)\mingw-w64\i686-8.1.0-posix-dwarf-rt_v6-rev0\mingw32\bin;C:\Program Files (x86)\Windows Kits\8.1\Windows Performance Toolkit\;C:\Program Files\Microsoft SQL Server\110\Tools\Binn\;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;C:\Program Files\nodejs\;C:\Program Files (x86)\Yarn\bin\;C:\WINDOWS\System32\OpenSSH\;C:\Program Files\Git\cmd;C:\Users\rajagopalan.m\.cargo\bin;C:\allure-framework\allure-2.9.0\bin;C:\Users\rajagopalan.m\AppData\Local\Microsoft\WindowsApps;C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\Tools\vsdevcmd\ext;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;C:\Users\rajagopalan.m\AppData\Roaming\npm;C:\Users\rajagopalan.m\AppData\Local\Yarn\bin;C:\Users\rajagopalan.m\AppData\Local\GitHubDesktop\bin;C:\Program Files (x86)\Google\Chrome\Application;C:\Ruby26\bin;C:\Program Files\ImageMagick-7.0.10-Q16-HDRI;C:\Users\rajagopalan.m\AppData\Local\Microsoft\WindowsApps;
        C:/Ruby26/lib/ruby/gems/2.6.0/gems/streamio-ffmpeg-3.0.2/lib/streamio-ffmpeg.rb:105:in `which'
        C:/Ruby26/lib/ruby/gems/2.6.0/gems/streamio-ffmpeg-3.0.2/lib/streamio-ffmpeg.rb:59:in `ffprobe_binary'
        C:/Ruby26/lib/ruby/gems/2.6.0/gems/streamio-ffmpeg-3.0.2/lib/ffmpeg/movie.rb:28:in `initialize'
        C:/Ruby26/lib/ruby/gems/2.6.0/gems/screen-recorder-1.4.0/lib/screen-recorder/common.rb:103:in `new'
        C:/Ruby26/lib/ruby/gems/2.6.0/gems/screen-recorder-1.4.0/lib/screen-recorder/common.rb:103:in `prepare_video'
        C:/Ruby26/lib/ruby/gems/2.6.0/gems/screen-recorder-1.4.0/lib/screen-recorder/common.rb:38:in `stop'
        C:/A/RubyAutomation_POM/FrameWork/Elements/Elements.rb:115:in `<top (required)>']
```

**Comment by kapoorlakshya** [comment]
Ah, yeah, `ffprobe` is used to collected the file metadata when the recorder is stopped. You can pass a custom path for `ffprobe` like this:

```ruby
FFMPEG. ffprobe_binary = '/path/to/ffprobe.exe'
```

**Comment by Rajagopalan-M** [comment]
That's great! thanks.

**Comment by kapoorlakshya** [comment]
`ScreenRecorder.ffprobe_binary` setter and getter methods will soon be available in version 1.4.1.
